docs/introduction/using-form-types.md

```diff
@@ -194,6 +194,26 @@ Required option that needs to be string or null and is a text that is shown unde
 
 Defines which entity should the files be assigned to.
 
+### [MultiLocaleFileUploadType](https://github.com/shopsys/shopsys/blob/master/packages/framework/src/Form/MultiLocaleFileUploadType.php)
```

Comment:
the link to `master` will not work, I would put `15.0` version instead here (and in the other links)

Reply:
fixed in my added lines, but there are mutiple others

Reply:
I have created an issue :wink: https://github.com/shopsys/shopsys/issues/3294